cluster/examples/kubernetes/ceph/cluster-on-pvc.yaml

```diff
@@ -60,8 +60,7 @@ spec:
       # across nodes as much as possible. Unfortunately the pod anti-affinity breaks down
       # as soon as you have more than one OSD per node. If you have more OSDs than nodes, K8s may
       # choose to schedule many of them on the same node. What we need is the Pod Topology
-      # Spread Constraints, which is alpha in K8s 1.16. This means that a feature gate must be
-      # enabled for this feature, and Rook also still needs to add support for this feature.
+      # Spread Constraints.
       # Another approach for a small number of OSDs is to create a separate device set for each
       # zone (or other set of nodes with a common label) so that the OSDs will end up on different
       # nodes. This would require adding nodeAffinity to the placement here.
```

Comment:
How about adding a sample topology constraint in this yaml file? 
- Add a comment that k8s 1.18 or newer is required
- The sample constraint could be commented out since it may not apply in the common case, especially before 1.18 is widely used.

Reply:
Added, thanks.